src/containers/bitset.c

```diff
@@ -1013,6 +1014,10 @@ bool bitset_container_validate(const bitset_container_t *v, const char **reason)
         *reason = "cardinality is incorrect";
         return false;
     }
+    if (v->cardinality <= DEFAULT_MAX_SIZE) {
+        *reason = "cardinality is too small";
```

Comment:
Might be worthwhile to specify "too small for a bitmap container" or something, but as long as the string is unique, I suppose it'll lead us here anyway.

Reply:
Changed the message. Thanks.